src/routes/contracts/__tests__/get-contract.e2e-spec.ts

```diff
@@ -13,7 +14,7 @@ describe('Get contract e2e test', () => {
 
   beforeAll(async () => {
     const moduleRef = await Test.createTestingModule({
-      imports: [AppModule],
+      imports: [AppModule.register(configuration)],
```

Comment:
I understand that for non-e2e tests the configuration object should be provided to the `register()` function, as this configuration object points to the _test configuration_. But in e2e tests, why is it needed to provide the `configuration` argument? The default value in `app.module.ts` would point to the same file (`@/config/entities/configuration`) 🤔

Reply:
It's not needed. I've removed the reference in those tests in [d074350](https://github.com/safe-global/safe-client-gateway/pull/875/commits/d074350c56820d558c42820b9bd0ca5067a4d9cc).